flask_app/blueprints/views.py

```diff
@@ -67,4 +67,10 @@ def summary():
 
 @views.route("/combadge")
 def get_combadge():
-    return send_file("../webapp/dist/assets/combadge.py")
+    combadge_type = request.args.get('combadge_type', default='python')
```

Comment:
Combadge should be selected by version, not implementation languange. Refer to the "old" Python version as `v1` and the new Rust version as `v2`. This will also allow us to add newer versions in the future.